Makefile

```diff
@@ -707,12 +708,9 @@ bdist_mpkg: 3rd
 fetch-data:
 	echo "I: fetching data from datadb"
 	[ -e datadb ] || mkdir -p datadb
-	rsync $(RSYNC_OPTS) $(DATA_URI)/tutorial_data $(DATA_URI)/mnist \
-		$(DATA_URI)/face_inversion_demo \
-	      	$(DATA_URI)/hyperalignment_tutorial_data \
-                $(DATA_URI)/haxby2001 \
-		datadb/ 
-	@for ds in datadb/*; do \
+	rsync $(RSYNC_OPTS) $(DATA_HOST)::'$(DATA_DIR)/tutorial_data $(DATA_DIR)/mnist $(DATA_DIR)/face_inversion_demo $(DATA_DIR)/hyperalignment_tutorial_data $(DATA_DIR)/haxby2001' \
```

Comment:
``> Damn phone``
Isn't that a phone that beats iphone? :-)

I tried multi-lined, but did not find a solution. When using::

    'a \
    b'

this is converted to 'a \b', and I did not find a way to get rid of the backslash. If you have any suggestion on how to do this I would really appreciate it.